layouts/partials/footer.html

```diff
@@ -1,22 +1,395 @@
+    }
+
+    .footer-list-doc li,
+    .footer-list-blog li,
+    .footer-list-com li {
+      margin-bottom: 1px !important;
+    }
+
+    .footer-list-doc li a,
+    .footer-list-blog li a,
+    .footer-list-com li a {
+      color: #b5b5b5;
+      font-size: 10px;
+      font-weight: 500;
+      line-height: normal;
+      text-decoration: none;
+    }
+
+    .footer-list-doc li a:hover,
+    .footer-list-blog li a:hover,
+    .footer-list-com li a:hover {
+      color: #c2c2c2;
+    }
+
+    .f-l-block {
+      display: flex;
+      flex-direction: column;
+    }
+
+    .f-logo {
+      padding-left: 50px;
+    }
+
+    .f-cr {
+      padding-right: 20px;
+      margin-top: 20px;
+    }
+
+    .f-terms {
+      display: flex;
+      flex-direction: row;
+      padding-left: 35px;
+      margin-top: 20px;
+    }
+
+    .f-l-logos {
+      margin-left: 70px;
+      margin-top: 30px;
+    }
+  }
+</style>
+
+<footer class="m-footer">
+  <div class="footer-list-doc">
+    <h4 class="doc">Document</h4>
+    <ul>
+      <li>
+        <a href="https://notaryproject.dev/docs/overview/">Project Overview</a>
+      </li>
+      <li>
+        <a href="https://notaryproject.dev/docs/quickstart/">Quickstart</a>
+      </li>
+      <li>
+        <a href="https://notaryproject.dev/docs/installation/"
+          >Installation Guides</a
+        >
+      </li>
+      <li><a href="https://notaryproject.dev/docs/tutorials/">Tutorials</a></li>
+      <li>
+        <a href="https://notaryproject.dev/docs/how-to/">How-to guides</a>
+      </li>
+      <li><a href="https://notaryproject.dev/docs/concepts/">Concepts</a></li>
+      <li>
+        <a href="https://notaryproject.dev/docs/troubleshooting/"
+          >Troubleshooting</a
+        >
+      </li>
+      <li><a href="https://notaryproject.dev/docs/faq/">FAQs</a></li>
+    </ul>
   </div>
-  <div class="row text-center text-white small">
-    <div class="col-12 text-center py-2 order-sm-2">
-      <a href="https://www.linuxfoundation.org/terms" target="_blank" rel="noopener">Terms</a> |
-      <a href="https://www.linuxfoundation.org/privacy" target="_blank" rel="noopener">Privacy</a> |
-      <a href="https://www.linuxfoundation.org/trademark-usage" target="_blank" rel="noopener">Trademarks</a> |
-      <a href="{{ .Site.Params.github_repo }}/blob/{{ .Site.Params.github_branch }}/LICENSE" target="_blank" rel="noopener">License</a>
-    </div>
+
+  <div class="footer-list-blog">
+    <h4>Blogs</h4>
+    <ul>
+      <li><a href="https://notaryproject.dev/blog/">Posts in 2023</a></li>
+      <li><a href="https://notaryproject.dev/blog/">Posts in 2022</a></li>
+      <li><a href="https://notaryproject.dev/blog/">Posts in 2021</a></li>
+    </ul>
+  </div>
+
+  <div class="footer-list-com">
+    <h4>Communities</h4>
+    <ul>
+      <li>
+        <a href="https://app.slack.com/client/T08PSQ7BQ/CQUH8U287/">Slack</a>
+      </li>
+      <li>
+        <a href="https://hackmd.io/_vrqBGAOSUC_VWvFzWruZw"
```

Comment:
Please make sure all external links will be opened and directed to a separate window.